jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -303,7 +324,19 @@
         for tenant in api.config.tenants.tenants
         if std.objectHas(tenant, 'mTLS')
       ] + [
-        tenant
+        {
+          id: tenant.id,
```

Comment:
why are we explicitly re-creating the structure of the tenant type vs rendering the tenant directly? Note, hidden fields (fields specified with `::`) won't be rendered in the final output.

Reply:
I guess I need to stop resisting and finally learn jsonnet...